build.sbt

```diff
@@ -3021,7 +3020,8 @@ buildEngineDistribution := {
   val _ = (`engine-runner` / assembly).value
   updateLibraryManifests.value
   val modulesToCopy = componentModulesPaths.value.map(_.data)
-  val engineModules = Seq(file("runtime.jar"))
+  val arrow = Seq((`runtime-language-arrow`/Compile/packageBin ).value)
```

Comment:
I think this is correct. Let's not assemble `runtime-language-arrow`, as that is unnecessary.